packages/tldraw/src/lib/shapes/frame/FrameShapeTool.ts

```diff
@@ -12,8 +12,11 @@ export class FrameShapeTool extends BaseBoxShapeTool {
 		const bounds = this.editor.getShapePageBounds(shape)!
 		const shapesToAddToFrame: TLShapeId[] = []
 		const ancestorIds = this.editor.getShapeAncestors(shape).map((shape) => shape.id)
+		const pageId = this.editor.getCurrentPageId()
 
-		this.editor.getCurrentPageShapes().map((pageShape) => {
+		this.editor.getSortedChildIdsForParent(pageId).map((pageShapeId) => {
+			const pageShape = this.editor.getShape(pageShapeId)
+			if (!pageShape) return
```

Comment:
`getSortedChildIdsForParent` only returns the parent's immediate children, and I think that's preventing the frame capture thing from working within frames themselves

![Kapture 2024-02-23 at 10 49 47](https://github.com/tldraw/tldraw/assets/1242537/7bb01b67-fbfa-41bb-8b9f-9fe6d0ab023b)

I think you could fix this by doing `this.editor.getSortedChildIdsForParent(shape.parentId)` instead?